tests/network/vmi_networking.go

```diff
@@ -1073,6 +1080,15 @@ sockfd = None`})
 	})
 })
 
+func waitForPrimaryAddress(vmi *v1.VirtualMachineInstance) bool {
+	virtClient, err := kubecli.GetKubevirtClient()
+	Expect(err).ToNot(HaveOccurred())
+
+	vmi, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Get(vmi.Name, &metav1.GetOptions{})
+	Expect(err).ShouldNot(HaveOccurred())
+	return len(vmi.Status.Interfaces) > 0 && vmi.Status.Interfaces[0].IP != ""
```

Comment:
So you could have just put the Eventually here so the function name will make sense.